example/client/main.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/cloudwego/kitex-examples/kitex_gen/api"
+	"github.com/cloudwego/kitex-examples/kitex_gen/api/echo"
+	"github.com/cloudwego/kitex/client"
+	"github.com/cloudwego/kitex/pkg/klog"
+	fileclient "github.com/kitex-contrib/config-file/client"
+	"github.com/kitex-contrib/config-file/monitor"
+)
+
+const (
+	filepath    = "kitex_client.json"
+	key         = "ClientName/ServiceName"
+	serviceName = "ServiceName"
+	clientName  = "ClientName"
+)
+
+func main() {
+	klog.SetLevel(klog.LevelDebug)
+
+	watcher, err := monitor.NewConfigMonitor(monitor.Options{
```

Comment:
- Maybe it's better to create a FileWatcher here instead of a ConfigMonitor, so it can be shared across clients (or not, depending on the user)
- The Monitor could be initialized within the clientSuite.